cmd/expiration-mailer/main.go

```diff
@@ -98,9 +98,23 @@ func (m *mailer) sendNags(contacts []string, certs []*x509.Certificate) error {
 	}
 	domains = core.UniqueLowerNames(domains)
 	sort.Strings(domains)
-
 	m.log.Debug(fmt.Sprintf("Sending mail for %s (%s)", strings.Join(domains, ", "), strings.Join(serials, ", ")))
 
+	var subject string
+	if m.subject != "" {
+		// If there is a subject from the configuration file, we should use it as-is
+		// to preserve the "classic" behaviour before we added a domain name.
+		subject = m.subject
+	} else {
+		// Otherwise, when no subject is configured we should make one using the
+		// domain names in the expiring certificate
+		subject = fmt.Sprintf("Certificate expiration notice for domain %q", domains[0])
+		if len(domains) > 1 {
+			subject += fmt.Sprintf(" (and %d more)", len(domains)-1)
```

Comment:
small nit: `and n more` is _slightly_ ambiguous, what do you think about sticking 'names' on the end so it would be `and n more names`. You'd hit the plurality edge case when there is only one more but I think it would be a little clearer.